openslides/core/views.py

```diff
@@ -353,3 +359,19 @@ def check_permissions(self, request):
         if (self.action in ('create', 'update', 'destroy') and
                 not request.user.has_perm('core.can_manage_tags')):
             self.permission_denied(request)
+
+
+class UrlPatternsView(utils_views.APIView):
+    """
+    Returns a dictonary with all url patterns as json.
+    """
+    URL_KWARGS_REGEX = re.compile(r'%\((\w*)\)s')
+
+    def get_context_data(self, **context):
+        result = {}
+        url_dict = get_resolver(None).reverse_dict
+        for url_pattern in filter(lambda key: isinstance(key, str), url_dict.keys()):
+            url = url_dict[url_pattern][0][0][0]
```

Comment:
why should it use getlist? You mean I should split up the url getting in more then variable? I don't like it.

We will mostly use url_pattern in angular. when someone defines pattern twice, then he can not access his urls. So he knowns that it does not work.